decidim-core/spec/system/mobile_header_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "Mobile header" do
+  let(:organization) { create(:organization) }
+  let!(:user) { create(:user, :admin, :confirmed, organization:) }
+
+  before do
```

Comment:
lets add a context here you visit as a desktop browser, then any of the classes should be visible

Reply:
Done